storage/indexer/Cargo.toml

```diff
@@ -39,4 +42,5 @@ rand = { workspace = true }
 
 [features]
 default = []
+test = []
```

Comment:
we usually just use "fuzzing", or "testing" if you are diligent (generally I think it isn't worth the effort to distinguish the two)